docs/section10/10-7.md

```diff
@@ -0,0 +1,71 @@
+                - Set the COPnf_full_load_b variable using data_lookup: `COPnf_full_load_b = data_lookup(table_G3_5_2,size_category_b)`
+            This check directly below is because mismatches between expected and modeled cooling efficiency could be due to there being multiple hvac zones combined into a single thermal block for specific system types.
+            - Check if system type is 3 or 4: `if hvac_sys_type_b in ["SYS-3", "SYS-3b", "SYS-4]:` 
+                - Set the system type 3 or 4 boolean variable to TRUE: `sys_type_3_or_4 = TRUE`      
+                - Check if the capacity bracket is the smallest: `if size_category_b == 65Btuh: smallcapBracket = TRUE`  
+                                                
+            **Rule Assertion:**
+            - Case 1: if COPnf_full_load_b equals full_load_efficiency_b and the number of cooling systems associated with the hvac system equals one and the system is not system type 3 or 4 then pass: `if COPnf_full_load_b == full_load_efficiency_b and num_cooling_sys == 1 and sys_type_3_or_4 == FALSE: PASS`
+            - Case 2: else if COPnf_full_load_b equals full_load_efficiency_b and the number of cooling systems associated with the hvac system equals one and the system is system type 3 or 4 and is in the smallest capacity bracket then pass: `elif COPnf_full_load_b == full_load_efficiency_b and num_cooling_sys ==1 and sys_type_3_or_4 == TRUE and smallcapBracket == TRUE: PASS`           
+            - Case 3: else if COPnf_full_load_b does not equal the full_load_efficiency_b and the number of cooling systems associated with the hvac system equals one and the system is system type 3 or 4 and is in the smallest capacity bracket then pass: `elif COPnf_full_load_b != full_load_efficiency_b and num_cooling_sys == 1 and sys_type_3_or_4 == TRUE and smallcapBracket == TRUE: FAIL`            
+            - Case 4: else if COPnf_full_load_b equals full_load_efficiency_b and the number of cooling systems associated with the hvac system equals one and the system is system type 3 or 4 and is in the smallest capacity bracket then pass (in the raise_message please provide the hvac_b.id,COPnf_full_load_b, full_load_efficiency_b, and the total_cool_capacity_b): `if COPnf_full_load_b == full_load_efficiency_b and num_cooling_sys ==1 and sys_type_3_or_4 == TRUE and smallcapBracket == FALSE: UNDETERMINED raise_message "CHECK IF THE BASELINE EFFICIENCY WAS MODELED CORRECTLY BASED ON THE EQUIPMENT CAPACITY OF THE THERMAL BLOCK DIVIDED BY THE NUMBER OF HVAC ZONES OR RESIDENTIAL SPACES."`             
+            - Case 5: Else if the number of cooling systems associated with the hvac system is greater than 1 (in the raise_message please provide the hvac_b.id): `elif num_cooling_sys > 1: UNDETERMINED and raise_message "THE NUMBER OF COOLING SYSTEMS ASSOCIATED WITH THE BASELINE HVAC SYSTEM IS GREATER THAN ONE WHICH IS UNEXPECTED AND THEREFORE THE OUTCOME OF RULE IS UNDETERMINED. CONDUCT MANUAL CHECK."`
```

Comment:
I am not sure why we need to check if number of systems is greater than 1.

Reply:
Modify so it checks each separately.